fullscreen.bs

```diff
@@ -67,6 +67,14 @@ stated otherwise it is unset.
 <p>All <a for=/>documents</a> have an associated <dfn>list of pending fullscreen events</dfn>, which
 is an <a>ordered set</a> of (<a>string</a>, <a>element</a>) <a>tuples</a>. It is initially empty.
 
+<p>All <a for=/>documents</a> have an associated <dfn export>pending fullscreen request flag</dfn>.
```

Comment:
Let's rename this to "has pending fullscreen request" and make it a boolean. Should also maybe use `for=Document`?